### ISSUE TITLE
Integrate EPSS

### ISSUE THREAD

**Issue by Arcuza** [opencve/opencve#327]
**Is your feature request related to a problem?**
Managing vulnerabilities by severity is cumbersome and in many cases an impossible problem. There's a need to also have an Exploitation Predictibility capability when prioritizing which vulnerabilities to go for.

**Do you have a solution in mind, or a suggestion to improve OpenCVE?**
FIRST EPSS is a daily estimate of the probability of exploitation activity being observed over the next 30 days. It is designed from the ground up to make the best use of all of the information available. https://www.first.org/epss/model

By integrating FIRST's EPSS API, which returns EPSS score for a CVE, into OpenCVE users could make another slice and dice trough data, namely by EPSS score. And track changes in EPSS score for a vulnerability. https://api.first.org/epss/

This would be God-like for OpenCVE...

**Comment by pierreavn** [comment]
+1

**Comment by Whatsonyourmind** [comment]
@ncrocfer Fair point — apologies for the off-topic content. Glad to hear EPSS is already in the next release. Looking forward to it.
